AppDB/appscale/datastore/groomer.py

```diff
@@ -556,6 +557,15 @@ def clean_up_indexes(self, direction):
       start_key = ''
     end_key = dbconstants.TERMINATING_STRING
 
+    # Indicate that an index scrub has started after the journal was removed.
+    if direction == datastore_pb.Query_Order.ASCENDING and not start_key:
+      index_state = self.db_access.get_metadata(
+        cassandra_interface.INDEX_STATE_KEY)
+      if index_state is None:
+        self.db_access.set_metadata(
+          cassandra_interface.INDEX_STATE_KEY,
+          cassandra_interface.IndexStates.SCRUB_IN_PROGRESS)
```

Comment:
Is there a chance if a groomer dies (say out of memory) to have an index that will stay in this state forever?